ARM/ResourceGroupScope-ARM-Deployment.yml

```diff
@@ -0,0 +1,36 @@
+name: workflow to deploy an ARM Template to a resource group
+on: push
+
+# CONFIGURATION
+# For help, go to https://github.com/Azure/Actions
+
+env:
+  AZURE_RESOURCE_GROUP: << Azure Resource Group >>   # set this to your Azure Resource group's name  
+  AZURE_SUBSCRIPTION_ID: << Subscription Id >>   # set this to your Azure Subscription Id
+  
+jobs:
+  build-and-deploy-to-dev:
+    runs-on: ubuntu-latest
+    steps:
+                
+    # Authentication
+    # Set up the following secrets in your repository: AZURE_CREDENTIALS
+    # For details on usage of secrets, please refer https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets
+    - name: Azure Login
```

Comment:
Here also, we can specify the commapnd for spn creation (the way we have done in subscription template)

Reply:
we should not need to because the Azurelogin action talks about this in detail. and will be best update to place to capture this info. I added Subs scope because I expect login to be done mostly on RG or Resource level.